### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by BouaikIlham**
- [ ] Created an index.html file. 
- [ ] Created an empty To Do List placeholder(<div> or <ul> element). The index.html file  set template using the HTML Webpack Plugin.
- [ ] Created an index.js file and set an array of some simple to do tasks (array of objects). Each task object  contain three keys:
      description [string].
      completed [bool].
      index: [number].
- [ ] WritedOn page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- [ ] Created a style.css and set rules for the To Do List
- [ ]  CSS All my source files (index.html, index.js and style.css)  located in /src directory, and my distribution files will generated by webpack and served by webpack dev server from /dist folder. 
- [ ] loaded by Webpack Style/CSS Loader.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "to-do-list-",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode production"
```
Comment (brytebee) [inline package.json]: - [x] **Optional:** The steps to run your code isn't specified and the general start script is missing. Kindly add `"start": "webpack serve --open"` to `package.json` script section.
- [x] **Optional** Kindly update your scripts with the below :
 `"build": "webpack",
    "watch": "webpack --watch",
    "start": "webpack serve --open"`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Cut the grass',
+    completed: false,
+    index: 3,
+  },
+];
+
```
Comment (brytebee) [inline src/index.js]: - [ ] **Required:** Great job implementing this. 👏 However, the To-do list should be rendered in the order of their index. Kindly sort by their indexes before displaying.
Consider this:
`Array.sort((a, b) => a.index - b.index);`

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @BouaikIlham👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- ✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @BouaikIlham, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**